tasks/go_test.py

```diff
@@ -752,6 +754,15 @@ def get_impacted_packages(ctx, build_tags=None):
         if file.endswith(".go") or file.endswith(".mod") or file.endswith(".sum")
     }
 
+    # Modification to go.mod and go.sum should force the tests of the whole module to run
+    for file in files:
+        if file.endswith(".mod") or file.endswith(".sum"):
```

Comment:
nit, it's probably not possible to have a `pro.mod` or `oppo.sum` but at least you are sure like this, wdyt?
```suggestion
        if file.endswith("go.mod") or file.endswith("go.sum"):
```

Reply:
Guess we could use `==` at this point

Reply:
I guess `file` is a full path, not just the filename, but I agree checking the file name would be better

Reply:
I will apply Nicolas suggestion, assuming no one will ever name its file `coucou.go.mod`